backend/src/test/java/org/example/backend/controller/ProjectControllerTest.java

```diff
@@ -53,7 +53,7 @@ void getAllProjectsTest() throws Exception {
     void saveNewProjectTest() throws Exception {
```

Comment:
Just a small suggestion. it would be more readable, if the name of the test describes, what you expect, when something happens. :) for example saveNewProjectTest_returnProjectwithId1..._whenProjectBlaBla..()